data/assets/examples/renderable/renderablemodel/advanced/model_shader.asset

```diff
@@ -0,0 +1,51 @@
+-- Custom Shaders
+-- This example loads a simple model with user custom shaders. The parent is not
+-- set which defaults the position of the model to the center of the Solar System.
```

Comment:
```suggestion
-- This example loads a simple model with custom shaders.
```

Since we don't set the Parent in any example, we don't need to mention it explicitly